src/style.css

```diff
@@ -0,0 +1,111 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+}
+
+html {
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+  font-family: "Roboto", sans-serif;
+  --link-color: #00acdc;
+  --hover-color: #fff;
+}
+
+.product {
+  width: 200px;
+  height: 406px;
```

Comment:
Don't add sizes to BEM-blocks. Use mix where necessary and move all external styles under element selector.
checklist 7 https://github.com/mate-academy/layout_product-cards/blob/master/checklist.md